build_push_minimal.sh

```diff
@@ -0,0 +1,15 @@
+# Publish inventory images to your personal quay.io repository
+# This script uses podman as a first class citizen and is compatible with MacOS for cross-compilation
+# Excludes redhat.registry.io as this should not be used for production-ready images
+set -exv
+
+if [[ -z "$QUAY_REPO_INVENTORY" ]]; then
+    # required since this script is not used in the CI pipeline, publishing should
+    # only happen from a developer's local machine to their personal repo
+    echo "QUAY_REPO_INVENTORY must be set"
+    exit 1
+fi
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+podman build --platform linux/amd64 --build-arg TARGETARCH=amd64 -t "${QUAY_REPO_INVENTORY}:${IMAGE_TAG}" -f ./Dockerfile
```

Comment:
I think we can use both docker and podman for this using the `scripts/check_docker_podman.sh`

Example here: https://github.com/project-kessel/inventory-api/blob/a85a24d48f9b4ce3130a4f06c7d6dd8cc12e82d0/scripts/start-inventory.sh#L4C1-L5C83

Reply:
Thanks for pointing this out Adam, updated.